sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -0,0 +1,251 @@
+from ._generated._phone_numbers_client import PhoneNumbersClient as PhoneNumbersClientGen
+from ._generated.models import PhoneNumberSearchRequest
+from ._shared.utils import parse_connection_str, get_authentication_policy
+from ._version import SDK_MONIKER
+import pytest
+
+
+class PhoneNumbersClient(object):
+    def __init__(
+        self,
+        endpoint, # type: str
+        credential, # type: str
+        **kwargs # type: Any
+    ):
+        # type: (...) -> None
+        try:
+            if not endpoint.lower().startswith('http'):
+                endpoint = "https://" + endpoint
+        except AttributeError:
+            raise ValueError("Account URL must be a string.")
+
+        if not credential:
+            raise ValueError(
+                "You need to provide account shared key to authenticate.")
+
+        self._endpoint = endpoint
+        self._phone_number_client = PhoneNumbersClientGen(
+            self._endpoint,
+            authentication_policy=get_authentication_policy(endpoint, credential),
+            sdk_moniker=SDK_MONIKER,
+            **kwargs)
+    
+    @classmethod
+    def from_connection_string(
+            cls, conn_str,  # type: str
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> PhoneNumbersClient
+        """Create PhoneNumbersClient from a Connection String.
+        :param str conn_str:
+            A connection string to an Azure Communication Service resource.
+        :returns: Instance of PhoneNumbersClient.
+        :rtype: ~azure.communication.PhoneNumbersClient
+        """
+        endpoint, access_key = parse_connection_str(conn_str)
+
+        return cls(endpoint, access_key, **kwargs)
+    
+    
+    @distributed_trace
+    def begin_purchase_phone_numbers(
+            self, 
+            search_id,
+            **kwargs
+    ):
+         # type: (...) -> LROPoller[None]
+        """Purchases phone numbers.
+
+        :param search_id: The search id.
+        :type search_id: str
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword polling: Pass in True if you'd like the LROBasePolling polling method,
+            False for no polling, or your own initialized polling object for a personal polling strategy.
+        :paramtype polling: bool or ~azure.core.polling.PollingMethod
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if no Retry-After header is present.
+        :rtype: ~azure.core.polling.LROPoller[~azure.communication.phonenumbers.models.PhoneNumberSearchResult]
+        """
+        return self._phone_number_client.phone_numbers.begin_purchase_phone_numbers(
+            search_id,
+            **kwargs
+        )
+    
+    @distributed_trace
+    def begin_release_phone_number(
+            self, 
+            phone_number, 
+            **kwargs
+    ):
+        # type: (...) -> LROPoller[None]
```

Comment:
When you use `(...)` in the type hint, the expectation is that the types are added in the method signature.
```
def begin_release_phone_number(
            self, 
            phone_number,   # type: str
            **kwargs
    ):
```

Reply:
Looks like these parameter type hints need to be added throughout the client.